lib/aescrypt.rb

```diff
@@ -74,10 +84,23 @@ def self.decrypt_data(encrypted_data, key, iv, cipher_type)
   #:arg: iv => String
   #:arg: cipher_type => String  
   def self.encrypt_data(data, key, iv, cipher_type)
-    aes = OpenSSL::Cipher::Cipher.new(cipher_type)
+    self.warn_if_ecb cipher_type
+
+    aes = OpenSSL::Cipher.new(cipher_type)
     aes.encrypt
     aes.key = key
     aes.iv = iv if iv != nil
     aes.update(data) + aes.final      
   end
+
+  private
+  def self.warn_if_ecb(cipher_type)
+    if cipher_type.downcase.include? 'ecb'
```

Comment:
And if it includes 'aes'.

Actually, ECB mode is not a good choice with any cipher :)